app/views/core_induction_programmes/mentor_materials/edit.html.erb

```diff
@@ -3,9 +3,6 @@
     <h1 class="govuk-heading-l">Content change preview</h1>
     <%= form_with model: @mentor_material, url: mentor_material_path, method: :put, id: "mentor-materials-update-form" do |f| %>
       <%= render "mentor_material_fields", locals = { f:f } %>
-      <%= f.govuk_submit "See preview" %>
```

Comment:
That probably means that mentor material controller can be simplified (as can some tests)

Reply:
The preview functionality was untested 👀